templates/company/contracts/CompanyTemplate.sol

```diff
@@ -48,24 +52,64 @@ contract CompanyTemplate is BaseTemplate {
     }
 
     function newInstance(string _id, address[] _holders, uint256[] _stakes, uint64[3] _votingSettings, uint64 _financePeriod, bool _useAgentAsVault) public {
+        _verifyCompanyParameters(_holders, _stakes, _votingSettings);
+        (Kernel dao, ACL acl) = _createDAO();
+        (, Voting voting) = _setupApps(dao, acl, _holders, _stakes, _votingSettings, _financePeriod, _useAgentAsVault);
+        _transferRootPermissionsFromTemplate(dao, voting);
+        _registerID(_id, dao);
+    }
+
+    function newInstance(
```

Comment:
We should add a comment to `newInstanceAndToken()` noting that we can't fit the payroll app in a single block with it (I'm seeing that at 6.9m and the delta for including the Payroll app at 1.1m?)